src/well_known_did_config.rs

```diff
@@ -1,3 +1,38 @@
+//! This module contains the structs and functions to parse the well-known DID configuration file and also produce the well-known DID configuration file.
```

Comment:
Do you think this should be a script we can run for the user?

Reply:
Bundling it inside the rust code stops you from generating a well known. How do we want to do this when we actually write this to chain?

Reply:
Not sure how to handle that. Longterm we would have a CLI tool for that

Reply:
All good as discussed.